lld/test/ELF/arm-adr-pic.s

```diff
@@ -0,0 +1,13 @@
+// REQUIRES: arm
```

Comment:
Instead of a new file, you can reuse an existing file. You can add a `-shared` RUN line to `arm-adr.s` with a comment stating that `/// R_ARM_ALU_PC_G0 referencing a non-preemptible symbol can be used in PIC links.'

Reply:
Thanks, that's a good idea -> DONE